docs/Webpack.md

```diff
@@ -145,7 +161,7 @@ Now that Jest knows how to process our files, we need to tell it how to _find_ t
 
 > Note: `<rootDir>` is a special token that gets replaced by Jest with the root of your project. Most of the time this will be the folder where your `package.json` is located unless you specify a custom `rootDir` option in your configuration.
```

Comment:
Perhaps this update could be a good excuse to add few missing admonitions? (;

```suggestion
::: note

`<rootDir>` is a special token that gets replaced by Jest with the root of your project. Most of the time this will be the folder where your `package.json` is located unless you specify a custom [`rootDir`](Configuration.md#rootdir-string) option in your configuration.

:::
```